packages/core/src/page/renderPageContent.ts

```diff
@@ -74,6 +74,6 @@ export const renderPageContent = ({
       'frontmatter'
     ),
     sfcBlocks,
-    title: frontmatter.title ?? title,
+    title: isString(frontmatter.title) ? frontmatter.title : title,
```

Comment:
Maybe

```
frontmatter.title ? `${frontmatter.title}` : title
```

Reply:
Emm, then why not `String(fromtmatter.title)`

Reply:
As I remember, string template has better performance than string constructor in most cases.

The only difference is that string constructor can handle symbol, while string template cannot. But I think frontmatter.title is impossible to be a symbol value in this case.